inbm-lib/inbm_lib/constants.py

```diff
@@ -40,10 +40,6 @@
 # Path to inbm version files
 INBM_VERSION_FILE = get_canonical_representation_of_path(
     '/usr/share/intel-manageability/inbm-version.txt')
-INBM_VISION_VERSION_FILE_HOST = get_canonical_representation_of_path(
-    '/usr/share/intel-manageability/inbm-vision-host-version.txt')
-INBM_VISION_VERSION_FILE_NODE = get_canonical_representation_of_path(
-    '/usr/share/intel-manageability/inbm-vision-node-version.txt')
 
 # CentOS version path
 CENTOS_VERSION_PATH = get_canonical_representation_of_path('/etc/centos-release')
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Maintainability$</h2>**
**Suggestion:** The removal of INBM_VISION_VERSION_FILE_HOST and INBM_VISION_VERSION_FILE_NODE constants should be justified in the PR description or comments. If these constants are no longer needed due to a change in the system's architecture or a refactoring decision, this should be clearly documented to avoid confusion. If this change might affect other parts of the system that reference these constants, a thorough search should be conducted to ensure no other code is broken by their removal.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
# If the removal of INBM_VISION_VERSION_FILE_HOST and INBM_VISION_VERSION_FILE_NODE is intentional and they are no longer used,
# ensure to document the reason for their removal and verify that no other part of the system is affected.
INBM_VERSION_FILE = get_canonical_representation_of_path(
    '/usr/share/intel-manageability/inbm-version.txt')
```